sda-download/dev_utils/compose.yml

```diff
@@ -105,6 +105,22 @@ services:
       - "8443:8443"
     restart: always
 
+  reencrypt:
+    image: ghcr.io/neicnordic/sensitive-data-archive:v0.3.7
```

Comment:
`:latest` instead (and provide the build context)?

Reply:
Hi @pontus, I checked but it seems the image `sensitive-data-archive` does not have a tag `latest`. Also, regarding the build context, I think it is better to be addressed when `sda-download` is merged into the `sda` folder.